test/test/jfr/Ttsp.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.jfr;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.io.BufferedOutputStream;
+import java.io.FileOutputStream;
+
+class Ttsp {
```

Comment:
This example needs some comments. How does it expose long TTSP issue? Why getAllStackTraces() is called in a loop? Writing 1GB files in a loop may not be a good thing to do in a test - IO is not what we are testing here. Furthermore, not all systems may even have a spare gigabyte in /tmp volume.

Reply:
Thank you, I adapted this example from an online article regarding debugging ttsp issues, will rewrite with explanation